packages/next-common/lib/button/index.jsx

```diff
@@ -52,7 +53,16 @@ export default function _Button(props) {
       )}
     >
       {loading ? (
-        <SystemLoading className={cn(smallSize && "w-4 h-4")} />
+        loadingText ? (
+          <>
+            <span className="inline-flex mr-1">
+              <SystemLoading className={cn(smallSize && "w-4 h-4")} />
+            </span>
+            {loadingText}
+          </>
+        ) : (
+          <SystemLoading className={cn(smallSize && "w-4 h-4")} />
+        )
```

Comment:
Please confirm this modification with @2nthony . I'm not sure whether this change should be implemented in `button` component. I think we can implement another alternative button on higher business components.